docs/guide/reactivity.md

```diff
@@ -192,6 +196,12 @@ The following app will display any color you type in to the input. Try it with a
 
 The color is parsed in `on_input_submitted` and assigned to `self.color`. Because `color` is reactive, Textual also calls `watch_color` with the old and new values.
 
+!! warning
```

Comment:
I don't think this is worthy of a warning. Suggest we promote this to another paragraph, with a new sub-heading.